build.gradle

```diff
@@ -13,6 +13,7 @@ ext {
 
 group = 'com.iexec.blockchain'
 sourceCompatibility = '11'
+targetCompatibility = '11'
```

Comment:
Just a question, did you have any issues without it?

Reply:
Nope, but I saw it was defined in all projects.
Should I revert it ?

Reply:
No it's great, I was just wondering :+1: